af-plans/juiceshop-selenium-auth/js-auth.bat

```diff
@@ -0,0 +1,7 @@
+
+set JS_USER=test@test.com
+set JS_PWD=test123
+
+# Customize the paths to the zap.bat file and the yaml file to match your environment
```

Comment:
While not wrong as is I think we should call it `automation plan` instead of `yaml file`.